projects/lib/package.json

```diff
@@ -4,13 +4,13 @@
   "author": {
     "name": "Manfred Steyer"
   },
-  "version": "8.0.1",
+  "version": "9.0.0-RC1",
   "repository": "manfredsteyer/angular-oauth2-oidc",
   "dependencies": {
     "jsrsasign": "^8.0.12"
   },
   "peerDependencies": {
-    "@angular/common": ">=6.0.0",
-    "@angular/core": ">=6.0.0"
+    "@angular/common": ">=9.0.0",
+    "@angular/core": ">=9.0.0"
```

Comment:
This will exclude older angular versions. should probably be as it was unless the new update introduces breaking changes for older versions

Reply:
True! I did do it intentionally but without too much confidence I guess: since there were so many migrations and compiler changes in NG9 (more so than 7 and 8), and since there's limited capacity/enthusiasm in this lib's community to support mixing newer versions of the lib with older versions of Angular.... I guessed that this change may be a good one.

But I suppose we can bring it down again, and consider using v9 of the lib with <v9 angular versions a "no warranty" thing?

Leaving this choice up to Manfred I think.